src/pkg/packaging/osx/sharedframework/shared-framework-distribution-template.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8" standalone="no"?>
 <installer-gui-script minSpecVersion="1">
     <title>{SharedFxBrandName} (x64)</title>
-    <license file="eula.rtf" mime-type="application/rtf" />
```

Comment:
We want to show MIT License link instead of Eula rtf.
See  https://github.com/dotnet/core-setup/issues/2513